feat(clayui.com): Improve component docs by updating content Pt.2 (#3235)
liferay/clay: packages/clay-slider/README.mdx

sibling: 'docs/components/css-slider.html'
---

import {DecadeSlider, Slider} from '$packages/clay-slider/docs/index';

<div class="nav-toc-absolute">
<div class="nav-toc">

-   [Customization](#customization)
-   [API](#api)

</div>
</div>

> You can use Slider with just 2 props for it's basic usage, `value` and `onValueChange` which is used to set the value of the Slider.

Reviewer: ```suggestion
Slider is a controlled component and needs just 2 props for its basic use, `value` and `onValueChange`.
```

The props should be self explanatory for what they do, so no need to say "they are used to set the value"

<Slider />

> ## Customization

Reviewer: I think we should aim for specific rather than general for our headings. So this could be `Range and Step`

> For a more specific usage you can specify other props, like `min` and `max` to determine the range, and `step` to specify how much the value changes.

Reviewer: ```suggestion
For a more specific use case you can specify other props, like `min` and `max` to determine the range, and `step` to specify how much the value changes.
```
